docs/baking.md

```diff
@@ -0,0 +1,159 @@
+(you can read more about `tezos-node` history modes [here](https://tezos.gitlab.io/user/history_modes.html#history-modes)).
+
+Snapshot import requires `tezos-node` data directory doesn't contain `context` and `store` folders.
+Remove them before the snapshot import.
+
+In order to import the snapshot run the following command:
+```
+sudo -u tezos-node snapshot import --data-dir /var/lib/tezos/node-<network>/ <path to the snapshot file>
+```
+
+In case you're getting the error similar to
+```
+tezos-node: Error:
+              Invalid block BLjutMj47caB
+                Failed to validate the economic-protocol content of the block: Error:
+                                                                                Invalid signature for block BLjutMj47caB. Expected: tz1Na5QB98cDA.
+```
+
+You should init/update config in the data directory to match the desired network
+before importing the snapshot. To do the run the following:
+```
+sudo -u tezos tezos-node config init --data-dir /var/lib/tezos/node-<network> --network <network>
+#or
+sudo -u tezos tezos-node config update --data-dir /var/lib/tezos/node-<network> --network <network>
+```
+
+### Starting the node
+
+After the snapshot import, you can finally start the node by running
+```
+sudo systemctl start tezos-node-<network>.service
+```
+
+Note that even after the snapshot import node can still be out of sync, it may require
+some additional time to completely bootstrap.
+
+To stop the node run:
+```
+sudo systemctl stop tezos-node-<network>.service
+```
+
+You can check node logs via `journalctl`:
+```
+journalctl -u tezos-node-<network>.service
+```
+
+## Setting up baker and endorser daemons
+
+Once you have fully synced local `tezos-node`, you can start baker and endorser daemons.
+
+Data directories for baker and endorser daemons are defined in the
+`/etc/default/tezos-baker-<proto>` and `/etc/default/tezos-endorser-<proto>`. Make
+sure to point them at the same directory to have the same set of known keys, e.g.
+point both `DATA_DIR`s to `/var/lib/tezos/baker`. You should also
+update `NODE_DATA_DIR` in the `/etc/default/tezos-baker-<proto>` to point at the desired
+node data directory, e.g. `/var/lib/tezos/node-<network>`.
+
+Create the aformentioned daemons data directory:
+```
+sudo -u tezos mkdir -p /var/lib/tezos/baker
+```
+
+Import you baker secret key to the data directory. There are two ways to import
+such key:
+1) You know either unencrypted or password-encrypted secret key for your address. In order
+to import such key run:
+```
+sudo -u tezos tezos-client -d /var/lib/tezos/baker import secret key <alias> <secret-key>
+```
+2) Secret key is stored on ledger. Open Tezow Wallet app on your ledger and run the following
+to import the key:
+```
+sudo -u tezos tezos-client -d /var/lib/tezos/baker import secret key <alias> <ledger-url>
+```
+Apart from importing the key, you'll also need to set it up for baking. Open Tezos Baking app
```

Comment:
From the rendered doc it isn't quite clear if what follows is needed in both cases or not.

In fact I'd suggest to divide this whole "Setting up baker and endorser daemons" section better into sub-steps(I suppose with `h3`/`###`:
- Set up daemons data directory
- Import you baker secret key to the data directory
- Start the daemons